app/Services/GoCardlessBankData.php

```
@@ -92,13 +93,17 @@ private function getAccessToken(): string
      * Parses the token response, updates access and refresh tokens, and sets their expiration times.
      *
-     * @param  mixed  $response  The HTTP response containing token data.
+     * @param  \Illuminate\Http\Client\Response  $response  The HTTP response containing token data.
      * @return string The new access token.
      */
-    private function processTokenResponse($response): string
+    private function processTokenResponse(\Illuminate\Http\Client\Response $response): string
     {
         $data = $response->json();
 
+        if (! isset($data['access'], $data['refresh'], $data['access_expires'], $data['refresh_expires'])) {
```

PHPStan (2.1.15)

Cannot access offset 'access' on mixed.


(offsetAccess.nonOffsetAccessible)